rxkeystore/src/androidTest/java/com/nexenio/rxkeystore/KeyAgreementTest.java

```diff
@@ -28,7 +28,7 @@ public void generateSecretKey_matchingKeyPairs_sameSecretKey() throws Exception
 
     private byte[] generateSecret(PrivateKey privateKey, PublicKey publicKey) throws NoSuchAlgorithmException, InvalidKeyException {
         KeyAgreement keyAgreement = KeyAgreement.getInstance("DH");
-        keyAgreement.init(privateKey);
+        keyAgreement.init(privateKey); // TODO: 03.06.21 DHPublicKey not for this KeyAgreement! on sdk 21 and 22
```

Comment:
Should we do something about this?

Reply:
Probably, please create a new ticket for that